### DISCUSSION TITLE
How to update a batch of Rows with different ID's and Data.

### DISCUSSION THREAD

**Discussion by ebaer-payix** [prisma/prisma#6604] [Q&A] [answered]
Hi,

I am trying to prevent having to do 100s or perhaps more individual record updates for obvious performance reasons.

I am wanting to bulk update a bunch of records that have different ID's which have had different Column values changed. (ID's are not changed)
ie:  I get a bunch of records from the database with a findMany(), then update the values on several records with various changes, then do a bulk update with all the changed records.

Is this possible, and if so, how do you code it ?

Thanks.

**Comment by pr0ph0z** [comment, marked as answer]
I think this is achievable using the basic `updateMany()` method but you need to do some object manipulations. It would be great if you can include the example schema and example of the records 😀

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
